### PR TITLE
removes gamelayer, finally got rgb working

### PR DESCRIPTION
**Pull request by Waples**
## Description

Updates to my `waples` keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jj40/keymaps/waples/keymap.c [hunk 1]
```diff
@@ -6,15 +6,13 @@ extern keymap_config_t keymap_config;
 
 #define _QWERTY 0
```
Comment (fauxpark) [inline keyboards/jj40/keymaps/waples/keymap.c]: You should turn these layer defines into an enum so you don't have to keep renumbering them :)

**Comment by Waples** [comment]
:thinking: weird; in my branch I dont have these conflicts. But yeah :upside_down_face: ill fix it in console then ^^

File: keyboards/jj40/keymaps/waples/keymap.c (hunk 1 above)
Comment (Waples) [reply keyboards/jj40/keymaps/waples/keymap.c]: How do I do that? :P

Comment (drashna) [reply keyboards/jj40/keymaps/waples/keymap.c]: You'd use:
```c
enum layers {
    _QWERTY,
    _DVORAK,
    _LEFTY,
    _RIGHTY,
    _DUAL,
};
```

It defaults to 0, for the first layer, and automatically increments.

Comment (noroadsleft) [reply keyboards/jj40/keymaps/waples/keymap.c]:
```c
enum layers {
    _QWERTY,
    _DVORAK,
    _LEFTY,
    _RIGHTY,
    _DUAL,
}:
```

This is programmatically identical to the `#define`s you have now.

File: keyboards/jj40/keymaps/waples/keymap.c [hunk 2]
```diff
@@ -59,11 +50,11 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   _______, KC_F9, KC_F10, KC_F11, _______, _______, _______, _______, _______, _______, _______, _______ \
 ),
 
-[_DUAL] =  LAYOUT_ortho_4x12( \
-  RESET,   _______, _______, _______, _______, QWERTY,  GAME,    _______, _______, _______, BL_BRTG, RESET, \
-  _______, KC_MPRV, KC_MSTP, KC_MPLY, KC_MNXT, AG_NORM, AG_SWAP, _______, _______, _______, BL_TOGG, _______, \
-  _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, TG_NKRO, _______, _______, _______, _______, _______, \
-  _______, _______, _______, _______, _______, _______, DVORAK,  _______, _______, _______, _______, _______ \
+[_DUAL] =  KEYMAP( \
```
Comment (noroadsleft) [inline keyboards/jj40/keymaps/waples/keymap.c]: We changed the layout macro naming convention around June of last year, to `LAYOUT` with an optional suffix, depending on the keyboard's codebase.

You need to change Line 53 as follows:

```suggestion
[_DUAL] =  LAYOUT_ortho_4x12( \
```

Comment (Waples) [reply keyboards/jj40/keymaps/waples/keymap.c]: changed

**Comment by drashna** [comment]
Travis CI is failing to compile this board, because `BACKLIGHT_CUSTOM_DRIVER` is enabled, but not actually used.

**Comment by drashna** [comment]
Any update on this?

**Comment by Waples** [comment]
@drashna ;)

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**